docs/05-concepts/06-database/11-migrations.md

```diff
@@ -17,6 +17,8 @@ fields:
   name: String
 ```
 
+If you want to transition a manually managed table to then be managed by Serverpod you first need to set this flag to `true`. Then you have two options, either delete the old table you had created by yourself, and this will let Serverpod manage the schema from a clean state. However, this means that you would lose any data that was stored in the table. The second option is to make sure that the table schema matches the expected schema you have configured. This can be done by either manually making sure the schema aligns, or by creating a [repair migration](#creating-a-repair-migration) to get back into the correct state.
```

Comment:
Maybe the options are a bit more clear if you list them instead of describing them in the same block? Using a two point list or something?